### PR TITLE
ODC-7598: Added e2e tests for data source filter group

### PR DESCRIPTION
**Pull request by lokanandaprabhu**
**Story:**
https://issues.redhat.com/browse/ODC-7598

**Tests:**
Screenshot 2024-07-26 at 12 31 17 PM

### CODE REVIEW

File: integration-tests/cypress/features/pipelines/data-source-filter.feature [hunk 1]
```diff
@@ -0,0 +1,69 @@
+@pipelines @odc-7598
+Feature: Data source filter
+              As a user, I want to differentiate live and archived data in PipelineRun and TaskRun list pages.
+
+        Background:
+            To run the tests, Red Hat OpenShift Pipelines operator subscription to be updated to refer to main branch of console plugin until this feature is released in operator.
```
Comment (sanketpathak) [inline integration-tests/cypress/features/pipelines/data-source-filter.feature:6]: Add this as a comment
```suggestion
            # To run the tests, Red Hat OpenShift Pipelines operator subscription to be updated to refer to main branch of console plugin until this feature is released in operator.
```

**Comment by sanketpathak** [comment]
Tests run green:
Screenshot 2024-07-30 at 5 31 59 PM
/lgtm

**Review by vikram-raj — COMMENTED** [review]
/lgtm

**Pull request merged**